pages/settings/SettingsWallet.qml

```diff
@@ -91,6 +91,7 @@ Rectangle {
         }
 
         MoneroComponents.SettingsListItem {
+            enabled: leftPanel.progressBar.fillLevel == 100
```

Comment:
```suggestion
            enabled: appWindow.walletSynced
```

did you try that?

Reply:
Yes, but `appWindow.walletSynced` is not always correct. It is `true `when you change wallet restore height and the wallet is synchronizing again.

Reply:
I guess ideally that would be fixed, but we can merge this for now.